WordPress/Classes/ViewRelated/People/PersonViewController.swift

```diff
@@ -0,0 +1,154 @@
+// TODO: JLP May.3.2016. Implement Me
+    }
+    
+    
+    // MARK: - Outlets
+    @IBOutlet var gravatarImageView : UIImageView! {
+        didSet {
+            let placeholder = UIImage(named: "gravatar.png")
+            gravatarImageView.downloadImage(person.avatarURL, placeholderImage: placeholder)
+        }
+    }
+    
+    @IBOutlet var fullNameLabel : UILabel! {
+        didSet {
+            fullNameLabel.text = person.fullName
+            fullNameLabel.font = WPStyleGuide.tableviewTextFont()
+            fullNameLabel.textColor = WPStyleGuide.darkGrey()
+        }
+    }
+    
+    @IBOutlet var usernameLabel : UILabel! {
+        didSet {
+            usernameLabel.text = "@" + person.username
+            usernameLabel.font = WPStyleGuide.tableviewSectionHeaderFont()
+            usernameLabel.textColor = WPStyleGuide.wordPressBlue()
+        }
+    }
+    
+    @IBOutlet var roleCell : UITableViewCell! {
+        didSet {
+            roleCell.textLabel?.text = NSLocalizedString("Role", comment: "User's Role")
+            roleCell.detailTextLabel?.text = person?.role.description.capitalizedString
+            roleCell.accessoryType = canPromote ? .DisclosureIndicator : .None
+            roleCell.selectionStyle = canPromote ? .Gray : .None
+            roleCell.userInteractionEnabled = canPromote
+            WPStyleGuide.configureTableViewCell(roleCell)
+        }
+    }
+    
+    @IBOutlet var firstNameCell : UITableViewCell! {
+        didSet {
+            firstNameCell.textLabel?.text = NSLocalizedString("First Name", comment: "User's First Name")
+            firstNameCell.detailTextLabel?.text = person.firstName
+            WPStyleGuide.configureTableViewCell(firstNameCell)
+        }
+    }
+    
+    @IBOutlet var lastNameCell : UITableViewCell! {
+        didSet {
+            lastNameCell.textLabel?.text = NSLocalizedString("Last Name", comment: "User's Last Name")
+            lastNameCell.detailTextLabel?.text = person.lastName
+            WPStyleGuide.configureTableViewCell(lastNameCell)
+        }
+    }
+    
+    @IBOutlet var displayNameCell : UITableViewCell! {
+        didSet {
+            displayNameCell.textLabel?.text = NSLocalizedString("Display Name", comment: "User's Display Name")
+            displayNameCell.detailTextLabel?.text = person.displayName
+            WPStyleGuide.configureTableViewCell(displayNameCell)
+        }
+    }
+    
+    @IBOutlet var removeCell : UITableViewCell! {
+        didSet {
+            removeCell.textLabel?.text = NSLocalizedString("Remove User", comment: "Remove User. Verb")
+            WPStyleGuide.configureTableViewDestructiveActionCell(removeCell)
+            removeCell.hidden = !canRemove
+        }
+    }
+    
+    
+    // MARK: - Private Properties
+    private var isSomeoneElse : Bool {
+        return blog.account.userID != person.ID
+    }
+    
+    private var canPromote : Bool {
+// TODO: JLP May.3.2016. Implement!
```

Comment:
It looks to me like you could link ALL of these to whatever ticket is going to implement these.  I'll not point out the remaining ones since it'll get very redundant.